src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/feedback/feedback.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+import { CdFormGroup } from '~/app/shared/forms/cd-form-group';
+import { ActionLabelsI18n } from '~/app/shared/constants/app.constants';
+
+import _ from 'lodash';
+
+@Component({
+  selector: 'cd-feedback',
+  templateUrl: './feedback.component.html',
+  styleUrls: ['./feedback.component.scss']
+})
+export class FeedbackComponent {
+  title = 'Feedback';
+  project: any = ['dashboard', 'block', 'objects', 'file_system', 'ceph_manager', 'orchestrator', 'ceph_volume', 'core_ceph'];
+  tracker: any = ['bug', 'feature'];
+  // severity: any = ['minor', 'major', 'critical']
+
+  feedbackForm: CdFormGroup;
+  constructor(
+    public activeModal: NgbActiveModal,
+    public actionLabels: ActionLabelsI18n
+  ) {
+    this.createForm();
+  }
+
+  private createForm() {
+    this.feedbackForm = new CdFormGroup({
+      project: new FormControl('', Validators.required),
+      tracker: new FormControl('', Validators.required),
+      subject: new FormControl('', Validators.required),
+      description: new FormControl('', Validators.required),
+    });
+  }
+
+  get f() {
+    return this.feedbackForm.controls;
+  }
+
+  onSubmit() {
+    fetch('/ui-api/feedback', {
```

Comment:
Please use RxJS and observables here. As @nizamial09 mentioned we have a special section for those API services, you may get some hints there how to implement your service.

Reply:
resolved